OXRS-SHA-StateMonitor-ESP32-FW.ino

```diff
@@ -27,7 +27,7 @@
 #define FW_NAME       "OXRS-SHA-StateMonitor-ESP32-FW"
 #define FW_SHORT_NAME "State Monitor"
 #define FW_MAKER      "SuperHouse Automation"
-#define FW_VERSION    "3.9.1"
+#define FW_VERSION    "4.0.a2"

```

Comment:
I usually just use the current version with a suffix - e.g. `3.9.1-TEST` - for custom test builds I send Frank for testing. I think anything we commit to Github should be tested already and a proper version number. If there are bugs, no problems, we just cut a x.x.y release to fix it.